src/category_theory/limits/over.lean

```diff
@@ -113,17 +124,33 @@ def cones_equiv_inverse (B : C) {J : Type v} (F : discrete J ⥤ over B) :
       { dsimp,
         rw ← f.w j,
         refl }
-    end } }
+    end }, }
```

Comment:
Is there any reason for this change?

Reply:
I like to leave trailing commas essentially everywhere they are allowed,
1. for both structures and tactics, so one can add more lines without editing the previous
2. for tactics, to get to see the pleasing "goals accomplished" when putting your mouse after the cursor.

Both are very minor, and I agree do not warrant  being "fixed". I'll undo this one.

Reply:
```suggestion
    end } }
```